NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -447,9 +441,6 @@ class NeuroID private constructor(
             )
             return false
         }
-        application?.let {
```

Comment:
Were these intentionally removed? I thought we had to call them to get the updated data? I don't see a call replacing it

Reply:
Don't need to update getLocation() anymore. These will be called directly when the json request is generated getting the latest location that the GPS.